_targets/meta/meta

```diff
@@ -0,0 +1,18 @@
+name|type|data|command|depend|seed|path|time|size|bytes|format|repository|iteration|parent|children|seconds|warnings|error
+.__global__|object|87b8d4f266c27bf4|||||||||||||||
+algorithm_csv|stem|98f92607dd48318b|72b6696561a29259|2c530c1562a7fbd1|48686226|data-raw/algorithm.csv|t19893.417610553s|530f140998b50d1c|112|file|local|vector|||0.003||
+algorithm_rda|stem|66f7362503371611|299578725db47571|a73128224189f273|-930623318|/home/luke/Documents/organizations/steno-aarhus/osdc/data/algorithm.rda|t19893.4400342999s|a96add13dbb67aa3|376|file|local|vector|||0.166||
+column_names_to_lower|function|1f9ed89ec76f0ce4|||||||||||||||
+create_algorithm_data|function|260984c321b10f83|||||||||||||||
+create_variable_description_data|function|e6984763ad307bc4|||||||||||||||
+get_algorithm_logic|function|87ff06f2c9ce0900|||||||||||||||
+get_register_abbrev|function|a82c561691b651df|||||||||||||||
+get_required_variables|function|986ccd76798db1ba|||||||||||||||
+include_hba1c|function|dd23db245d8e630d|||||||||||||||
+register_as_md_header|function|1b9b0bb62cc1e264|||||||||||||||
+register_data_as_md_table|function|27e92012b850a984|||||||||||||||
+registers_as_md_table|function|ab56ce7262ba313b|||||||||||||||
+variable_description_csv|stem|900853341b756f57|2347307fa16d111b|2c530c1562a7fbd1|113481566|data-raw/variable-description.csv|t19845.4834059685s|75bbd1de9a7a9806|4683|file|local|vector|||0.005||
+variable_description_rda|stem|450aefe34ecbfb32|1742010767fda041|824677c99eb59944|-1262878967|/home/luke/Documents/organizations/steno-aarhus/osdc/data/variable_description.rda|t19893.4447566339s|5272409a13fc9df9|1550|file|local|vector|||0.496|Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.ℹ Please use register_name instead of .dataregister_name. Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.ℹ Please use register_abbrev instead of .dataregister_abbrev. Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.ℹ Please use variable_name instead of .datavariable_name. Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.ℹ Please use start_year instead of .datastart_year. Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.ℹ Please use end_year instead of .dataend_year. Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.ℹ Please use danish_description instead of .datadanish_description. Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.ℹ Please use english_description instead of .dataenglish_description|
```

Comment:
This tidyselect warning is ugly, and seems simple to address, if the warning message is accurate. Should we create an issue about it?

Reply:
Yes, this is fixed in #127 ☺️